governance/README.md

```diff
@@ -29,7 +29,7 @@ To be recognized with **Contributor** status, a community member must demonstrat
 
 ### Nomination
 
-A **Maintainer** requests the addition of a new **Contributor** in the private *Maintainers* channel in Zulip, listing the number of nontrival contributions by that person. A person may also request that a **Maintainer** nominates them and vouch for them. If no objections are raised within one week, the person is added to the **Contributors** team.
+A **Maintainer** requests the addition of a new **Contributor** in the private *Maintainers* channel in Zulip, listing the number of nontrival contributions by that person. A person may also request that a **Maintainer** nominates them and vouch for them. If no objections are raised within one week, the person is added to the **Contributors** team. Additionally, if four other maintainers express support before the week has elapsed then there is no need to wait out the remaining time.
```

Comment:
We have discussed about this in the [TSC call today](https://github.com/servo/project/pull/184), it seems it'd be nice to add a minimum of 1 or 2 days of waiting time before we can make someone a contributor, in case someone has any concern to raise. Otherwise a new contributor could be approved in a few hours, without allowing any chance for someone to say anything about that person.

Would you be fine adding something like that to the new proposed text?